junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedInvocationNameFormatter.java

```diff
@@ -247,22 +240,8 @@ public synchronized void append(ArgumentsContext context, StringBuffer result) {
 		}
 
 		private @Nullable Object[] makeReadable(@Nullable Object[] arguments) {
-			Format[] formats = messageFormat.getFormatsByArgumentIndex();
-			@Nullable
-			Object[] result = Arrays.copyOf(arguments, Math.min(arguments.length, formats.length), Object[].class);
-			for (int i = 0; i < result.length; i++) {
-				if (formats[i] == null) {
-					result[i] = truncateIfExceedsMaxLength(StringUtils.nullSafeToString(arguments[i]));
-				}
-			}
-			return result;
-		}
-
-		private @Nullable String truncateIfExceedsMaxLength(@Nullable String argument) {
-			if (argument != null && argument.length() > this.argumentMaxLength) {
-				return argument.substring(0, this.argumentMaxLength - 1) + ELLIPSIS;
-			}
-			return argument;
+			return Arrays.copyOf(arguments,
```

Comment:
```suggestion
			return replaceNonPrintableCharacters(Arrays.copyOf(arguments,
```
you suggest like this, right?
@sbrannen

i would like to impl. this issue. Seems possible with limited scope.